app.js

```diff
@@ -22,4 +22,24 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+  .then(response => res.render ('beers', {response}) )
+});
+// console.log(response);
+
+app.get('/random-beers', (req, res) => {
+  punkAPI.getRandom()
+  .then(response => res.render ('random-beers', {response}) )
+});
+
```

Comment:
Great job William! The only thing I noticed is the naming for the returning value of the promises (repsonse) is repetitive. You would like to give it a more descriptive name like "randomBeer" or "beersList" to make it more readable. Whenever you have some time, I suggest you to go through the bonuses :)